samples/dotnet-azurefunction/TransferEventBetweenTopics.cs

```diff
@@ -19,8 +19,8 @@ public static class TransferEventBetweenTopics
         /// </summary>
         [FunctionName("TransferEventBetweenTopics")]
         public static void Run(
-            [DaprTopicTrigger(Topic = "A")] CloudEvent subEvent,
-            [DaprPublish(Topic = "B")] out object pubEvent,
+            [DaprTopicTrigger("%PubSubName%", Topic = "A")] CloudEvent subEvent,
+            [DaprPublish("%PubSubName%", Topic = "B")] out object pubEvent,
```

Comment:
So this is the impactful part of the change. We need pubsub names to be specified, and we don't really have a way to infer them. Making it mandatory is what we already do for state stores, and elsewhere in the .NET SDK that we handle this.

Reply:
Users will have a relatively easy time absorbing this change, because it's a new required parameter.

Reply:
Update here...

`[DaprPublish(PubSubName = ..., Topic = ...)]` will use properties since you can specify the pubsub name in the individual events as well.